src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.big_bar {
+  margin: 20px 0;
+}
+
+::placeholder {
+  color: #3d4e61;
+  outline: none;
+}
+
+.input {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  width: 100%;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  font-family: inherit;
+  font-weight: 400;
+}
+
+.input_box_big {
+  background-position-x: 25px;
+  background-position-y: 25px;
+  background-size: 19px;
+  padding: 25px 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input:hover {
+  font-family: inherit;
+  font-weight: 400;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input_box_big:hover {
+  padding: 25px 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: inherit;
+  font-weight: 900;
```

Comment:
Checklist, last point
<img width="574" alt="image" src="https://user-images.githubusercontent.com/118751034/212738171-643d9ea8-7c20-41dd-90b4-9ee47f4365a2.png">
```suggestion
```

Reply:
Could you please explain what should I do? Why do I need to remove font weight from Focus?